### ISSUE TITLE
Why lower loss, but lower accuracy?

### ISSUE THREAD

**Issue by barleyyuan** [1adrianb/face-alignment-training#23]
I have implemented a PyTorch version of  2D-FAN training on 300W-LP through your training opts(e.g., learning rate, optimizer). I found that my trained model get much lower loss than yours, but lower accuracy than yours. So I am wondering if you have some novel initialization of parameters or other training tricks. Could you give me some advices? Thank you!
